src/translate.service.ts

```diff
@@ -396,13 +396,12 @@ export class TranslateService {
      * @returns string
      */
     public getBrowserLang(): string {
-        if(typeof window === 'undefined' || typeof window.navigator === 'undefined') {
+        let browserLang = this.getBrowserCultureLang();
+
+        if(browserLang === 'undefined') {
```

Comment:
I'd rather keep `typeof window === 'undefined'` because undefined can be defined (yeah, I know :D)